charts/incubator/youtrack/Chart.yaml

```diff
@@ -0,0 +1,30 @@
+annotations:
+  truecharts.org/SCALE-support: "true"
+  truecharts.org/catagories: |
+    - ProjectManagement
+apiVersion: v2
+appVersion: latest
+dependencies:
+  - name: common
+    repository: https://library-charts.truecharts.org
+    version: 11.1.2
+deprecated: false
+description: YouTrack is a project management tool that can be adapted to your
+  processes to help you deliver great products.
+home: https://truecharts.org/charts/incubator/youtrack
+icon: https://truecharts.org/img/hotlink-ok/chart-icons/youtrack.png
+keywords:
+  - youtrack
+  - ProjectManagement
+kubeVersion: ">=1.16.0-0"
+maintainers:
+  - email: info@truecharts.org
+    name: TrueCharts
+    url: https://truecharts.org
+name: youtrack
+sources:
+  - https://github.com/truecharts/charts/tree/master/charts/incubator/youtrack
+  - https://www.jetbrains.com/youtrack/
+  - https://hub.docker.com/r/jetbrains/youtrack/
+type: application
+version: 0.0.1
```

Comment:
Please update order and spacing according to other apps in stable.
This looks like it's pulled from an incubator app.